introduction.md

```diff
@@ -1,3 +1,6 @@
 H! my name is Stefano eadn i'm trying to do this funking course in the half on the night
 
 My favourite CodeSchol path is JavaScript.
+*Javascript
+*Git
+*Python
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?